docs/manual/src/udl/index.md

```diff
@@ -1,6 +1,9 @@
 # The UDL file
 
-This file defines which functions, methods and types are exposed to the foreign-language bindings.
+A UDL file allows you to define an interface externally from your Rust code.
+It defines which functions, methods and types are exposed to the foreign-language bindings.
```

Comment:
Perhaps a line or two here about the relationship of the UDL file itself to binding generators. Ie it *can* be used as input to binding generators but typically `--library` is used.

Reply:
would make a great followup PR!